src/plots/barPlot.ts

```diff
@@ -397,6 +397,24 @@ export module Plots {
       }
     }
 
+    /**
+     * Makes sure the extent takes into account the widths of the bars
+     */
+    protected _computeExtent(dataset: Dataset, accScaleBinding: Plots.AccessorScaleBinding<any, any>, filter: Accessor<boolean>): any[] {
```

Comment:
Could this logic have been added to `_extentsForProperty()` instead? That method is already `protected`.